engine/src/main/java/io/zeebe/engine/processing/bpmn/event/StartEventProcessor.java

```diff
@@ -39,51 +33,26 @@ public StartEventProcessor(final BpmnBehaviors bpmnBehaviors) {
   }
 
   @Override
-  public void onActivating(final ExecutableStartEvent element, final BpmnElementContext context) {
-    stateTransitionBehavior.transitionToActivated(context);
+  public void onActivate(final ExecutableStartEvent element, final BpmnElementContext context) {
+    final var activated = stateTransitionBehavior.transitionToActivated(context);
+    stateTransitionBehavior.completeElement(activated);
```

Comment:
I just followed the enhancement, https://github.com/zeebe-io/enhancements/blob/master/ZEP004-wf-stream-processing.md#create-workflow-instance 

> the event Sequence_Flow_Taken is produced by the command StartEvent:Complete_Element. There is no explicit command for the sequence flow (like Take_Sequence_Flow) because it can't be interrupted (i.e. no wait state)

But yes probably it can be done in one go. It can't be terminate anymore, which is currently the case. @saig0 any opinion?